spec/features/create_scan_spec.rb

```diff
@@ -55,7 +57,9 @@
 
       click_button 'Send request'
 
-      expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
+      s = ScansController.new
+
+      expect(s.illiad_query(Scan.last)).to eq('something')
```

Comment:
I feel like this particular assertion doesn't really belong here.  This is a unit test on a method of the ScansController, but the unit test is being executed in the context of a feature/integration test (not in the context of the ScansController spec).

A possible suggestion would be to change `something` in line 49 to be a url (e.g. http://test.com/ or http://example.com/) and then you can assert that the `current_url` (after clicking the `Send request` button) is what you set as the return value of the `illiad_query` method (effectively validating that we redirect to the result of that method).

It would then be nice to have unit test(s) in the scans_controller_spec that invokes the `illiad_query` method and actually asserts what it returns.  As of right now you there are no tests that validate anything about how the attributes of the scan object get translated into the URL string we send to Illiad.